### PR TITLE
[CPU] [ARM] JIT Floor

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [ARM] jit floor*

### Tickets:
 - *CVS-138572*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
1. Could you please add support for x64 jit emitter? Just want to avoid unaligned support between device wherever possible
2. Don't we need to update tests? Or floor is covered already?

**Comment by eshoguli** [comment]
> 1. Could you please add support for x64 jit emitter? Just want to avoid unaligned support between device wherever possible
> 2. Don't we need to update tests? Or floor is covered already?

Tests were added, thanks. CVS-140753 for X64 was created.

**Comment by dmitry-gorokhov** [comment]
> > 1. Could you please add support for x64 jit emitter? Just want to avoid unaligned support between device wherever possible
> > 2. Don't we need to update tests? Or floor is covered already?
> 
> Tests were added, thanks. [CVS-140753](https://jira.devtools.intel.com/browse/CVS-140753) for X64 was created.

I mean lets implement x64 emitter in bounds of this PR. Should be pretty straightforward to do that.

**Comment by eshoguli** [comment]
> I mean lets implement x64 emitter in bounds of this PR. Should be pretty straightforward to do that.

x64, supported

File: src/plugins/intel_cpu/src/cpu_types.cpp [hunk 1]
```diff
@@ -187,7 +187,11 @@ static const TypeToNameMap& get_type_to_name_tbl() {
         {"Ceiling", Type::Math},
         {"Cos", Type::Math},
         {"Cosh", Type::Math},
+#if defined(OPENVINO_ARCH_ARM)
         {"Floor", Type::Math},
+#else
+        {"Floor", Type::Eltwise},
+#endif
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/cpu_types.cpp:194]: Why we need any arch specific conditions there? I would expect Floor to have Eltwise type for any platform

File: src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp [hunk 2]
```diff
@@ -194,6 +195,9 @@ const std::map<utils::ActivationTypes, std::vector<std::vector<float>>>& activat
         {Exp,         {{}}},
         {Clamp,       {{-2.0f, 2.0f}}},
         {Elu,         {{0.1f}}},
+#if !defined(OPENVINO_ARCH_ARM)
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp:198]: Why it is added for non ARM platforms only?

File: src/plugins/intel_cpu/src/cpu_types.cpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/cpu_types.cpp:194]: `Floor` for ARM32 is part of `Math` operation, as before.

File: src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp (hunk 2 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp:199]: was added for any platform except ARM32

Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp]: done, thanks

File: src/plugins/intel_cpu/src/cpu_types.cpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/cpu_types.cpp]: done, thanks

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**